Framework/Geometry/inc/MantidGeometry/Rendering/GeometryTriangulator.h

```diff
@@ -61,10 +62,10 @@ class MANTID_GEOMETRY_DLL GeometryTriangulator {
   const std::vector<double> &getTriangleVertices();
   /// get a pointer to the 3x(NumberOFaces) integers describing points forming
   /// faces (p1,p2,p3)(p4,p5,p6).
-  const std::vector<int> &getTriangleFaces();
+  const std::vector<uint32_t> &getTriangleFaces();
 #ifdef ENABLE_OPENCASCADE
 private:
-  boost::shared_ptr<TopoDS_Shape>
+  std::unique_ptr<TopoDS_Shape>
```

Comment:
One effect of this change is that the default behaviour of the `GeometryTriangulator` class would have now changed. `GeometryTriangulator` should not longer be copyable via an ordinary copy-constructor. As this has not caused a compiler error, I presume that this class in not meant to be copied, although it would have been possible previously.

This is very minor, but would it be worth stating that to some extent in the header, or marking the copy and copy-assignment operators as deleted? This could even be done in a follow on PR if there are other areas of the code that would benefit from this.

Reply:
@OwenArnold  I deleted the copy constructor as part of [this](https://github.com/mantidproject/mantid/pull/21880) PR